test/SlowTests/Server/Documents/PeriodicBackup/PeriodicBackupSlowTests.cs

```diff
@@ -4005,6 +4005,151 @@ public async Task can_backup_and_restore_with_subscription(BackupType backupType
             }
         }
 
+        [Fact, Trait("Category", "Smuggler")]
```

Comment:
`RavenFact`

Reply:
let's make the tests for all databases (sharded & non-sharded)

Reply:
this is in 5.4 :)